METIS/docs/example_notebooks/LSS-YSO_model_simulation.ipynb

```diff
@@ -847,12 +853,20 @@
     "plt.figure(figsize=(12, 6))\n",
     "plt.plot(det_wave, result_extract, label=\"flux-calibrated simulated spectrum\")\n",
     "# The scaling for the source spectrum has been estimated by eye\n",
-    "plt.plot(src_wave, src_extract * 1500, label=\"input spectrum\")\n",
-    "plt.xlabel(r\"Wavelength [$\\mu$m]\")\n",
+    "# plt.plot(src_wave, src_extract * 1500, label=\"input spectrum\")\n",
+    "plt.plot(src_wave, src_extract * .42, label=\"input spectrum\")\n",
```

Comment:
Okay, this is a factor 3600. For some reason I cannot find the change that causes this. It's probably hiding in plain sight.

Reply:
`exptime` was previously overwritten with `std_rectified[0].header['EXPTIME']` (which is 1 s), but then used to rescale the rectified results of the 3600 s observation, resulting in a net change by a factor of ~3600. I can't seem to link to the line in the rich diff here, but anyway, that's what was happening. I now used `std_exptime` everywhere it was actually meant to be used, to avoid confusion.